go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -300,7 +300,7 @@ func watchForChanges(configPath string) error {
 				if !ok {
 					return
 				}
-				klog.Errorf("fsnotify error %v", err)
+				klog.Errorf("Fsnotify error %v", err)
```

Comment:
That's wrong (though it's also a terrible log message so being forced to rewrite it would probably be a good thing.)